apitools/base/py/credentials_lib_test.py

```diff
@@ -132,11 +132,11 @@ def testGetServiceAccount(self):
         creds = self._GetServiceCreds()
         opener = mock.MagicMock()
         opener.open = mock.MagicMock()
-        opener.open.return_value = six.StringIO('default/\nanother')
+        opener.open.return_value = six.BytesIO(b'default/\nanother')
         with mock.patch.object(six.moves.urllib.request, 'build_opener',
                                return_value=opener,
                                autospec=True) as build_opener:
-            creds.GetServiceAccount('default')
+            creds.GetServiceAccount(b'default')
```

Comment:
It looks like this might be at fault for the Python 3.5 regression mentioned in [this comment](https://github.com/google/apitools/pull/254#issuecomment-460305008).

Maybe something like this would help remedy the regression:
```
if six.PY2:
    creds.GetServiceAccount(b'default')
else:
    creds.GetServiceAccount('default')

Reply:
Marking this above comment "request changes", sorry I'm still a little new to the github workflow. :slightly_smiling_face: